cmd/gokeyless/initialize.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/cloudflare/cfssl/csr"
+	"github.com/cloudflare/cfssl/log"
+	"github.com/cloudflare/gokeyless/server"
+)
+
+type v4apiError struct {
+	Code    json.Number `json:"code,omitempty"`
+	Message string      `json:"message,omitempty"`
+}
+
+type initAPIResponse struct {
+	Success  bool              `json:"success,omitempty"`
+	Messages []string          `json:"messages,omitempty"`
+	Errors   []v4apiError      `json:"errors,omitempty"`
+	Result   map[string]string `json:"result,omitempty"`
+}
+
+var apiEndpoint = "https://apix.cloudflare.com/certificate"
```

Comment:
It is now. I think you're looking at an outdated diff.

Jacob H. Haven
CloudFlare Systems Engineer

On Wed, Oct 21, 2015 at 12:55 PM, jkroll-cf <notifications@github.com>
wrote:

> In cmd/gokeyless/initialize.go
> <https://github.com/cloudflare/gokeyless/pull/28#discussion_r42673431>:
>
> > +	"github.com/cloudflare/gokeyless/server"
> > +)
> > +
> > +type v4apiError struct {
> > +	Code    json.Number `json:"code,omitempty"`
> > +	Message string      `json:"message,omitempty"`
> > +}
> > +
> > +type initAPIResponse struct {
> > +	Success  bool              `json:"success,omitempty"`
> > +	Messages []string          `json:"messages,omitempty"`
> > +	Errors   []v4apiError      `json:"errors,omitempty"`
> > +	Result   map[string]string `json:"result,omitempty"`
> > +}
> > +
> > +var apiEndpoint = "https://apix.cloudflare.com/certificate"
>
> I would say make it a parameter/flag. More useful that way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudflare/gokeyless/pull/28/files#r42673431>.
>